packages/connect/src/device/Device.ts

```diff
@@ -112,28 +113,23 @@ export class Device extends TypedEmitter<DeviceEvents> {
 
     featuresNeedsReload = false;
 
-    acquirePromise?: ReturnType<Transport['acquire']> = undefined;
-    releasePromise?: ReturnType<Transport['release']> = undefined;
-
-    runPromise?: Deferred<void>;
+    // variables used in one workflow: acquire -> transportSession -> commands -> run -> keepTransportSession -> release
+    private acquirePromise?: ReturnType<Transport['acquire']>;
+    private releasePromise?: ReturnType<Transport['release']>;
+    private runPromise?: Deferred<void>;
+    private transportSession?: Session | null;
```

Comment:
should be public?

Reply:
they seem to be used privately, or at least some of them

Reply:
not now, but it will be used in future modules, i've rebased few parallel [branches](https://github.com/trezor/trezor-suite/blob/d4f90fcf6dcc893e17edf8ffc12a7c278baaaaf5/packages/connect/src/device/prompts.ts#L32) where i need transportSession